### ISSUE TITLE
Micronaut CLI 3.8.x and 3.7.x seem to be broken on Ubuntu 20 LTS

### ISSUE THREAD

**Issue by aadrian** [micronaut-projects/micronaut-starter#1608]
Micronaut CLI **3.8.x and 3.7.x** seem to be **broken on Ubuntu 20 LTS**.

The error message for all commands is:
```bash
$ mn --version
mn: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by mn)
mn: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by mn)

```
This error is independent of the JVM used (tried several combinations), all managed with SDKMAN.

On Ubuntu 20 LTS, the default GLIBC version is 2.31:
```bash
$ ldd --version
ldd (Ubuntu GLIBC 2.31-0ubuntu9.9) 2.31
```
**Notes:**
 - Micronaut CLI **3.6.x** seems to have no problem however !
 - Also on **Ubuntu 22 LTS**  all versions seem to work.

Thank you.

**Comment by ShadowBrook** [comment]
I meet the same problem

**Comment by aadrian** [comment]
Unfortunately the problem still persists even with the freshly released v3.8.5  :( .

**Comment by aadrian** [comment]
It issue seems to be fixed with the latest Micronaut v3.8.8 !

Thank you very much !
